storage/move_packages.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+"""
+Automate Package Moving
+
+Helper script to automate moving packages in the Archivematica Storage Service.
+"""
+
+from __future__ import print_function
+import argparse
+import json
+import logging
+import logging.config  # Has to be imported separately
+import os
+import requests
+from six.moves import configparser
+import sys
+
+# This project
+from common import utils
+
+THIS_DIR = os.path.abspath(os.path.dirname(__file__))
+sys.path.append(THIS_DIR)
+
+LOGGER = logging.getLogger('storage')
+
+CONFIG_FILE = None
+
+
+def get_setting(setting, default=None):
+    return utils.get_setting(CONFIG_FILE, 'storage', setting, default)
+
+
+def setup(config_file):
+    global CONFIG_FILE
+    CONFIG_FILE = config_file
+
+    # Configure logging
+    default_logfile = os.path.join(THIS_DIR, 'automate-storage.log')
+    logfile = get_setting('logfile', default_logfile)
+    utils.configure_logging(logfile)
+
+
+def get_first_eligible_package_in_location(ss_url, location_uuid):
+    """
+    Get first package in a location that has a status of either UPLOADED or MOVING.
```

Comment:
This should probably also check for VERIFIED, but it should be discussed further.